.github/workflows/gluafixer.yml

```diff
@@ -0,0 +1,63 @@
+name: Lint GLua
+
+on:
+  workflow_call:
+    inputs:
+      config:
+        type: string
+        required: false
+        default: ""
+        description: "A URL or path (relative to your project root) of the glualint config"
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: project
+
+      - name: Prepare GLuaFixer
+        run: |
+          cd $GITHUB_WORKSPACE
+          curl -s https://api.github.com/repos/FPtje/GLuaFixer/releases/latest \
+          | grep "browser_download_url.*linux" \
+          | cut -d : -f 2,3 \
+          | tr -d \" \
+          | wget -O glualint.zip -qi -
+
+      - name: Extract GLuaFixer
+        run: |
+          cd $GITHUB_WORKSPACE
+          unzip glualint.zip
+
+      - name: Get config file
+        run: |
+          GLUALINT_PATH=${{ inputs.config }}
+          if [[ "$GLUALINT_PATH" == "" ]];
+          then
+              echo "::warn:: No glualint config provided. Using defaults."
+              echo "{}" > "$GITHUB_WORKSPACE/.glualint.json"
+          else
+            if [[ "$GLUALINT_PATH" == http* ]];
+            then
+                echo "::info:: Given remote glualint path. Downloading..."
```

Comment:
Nit: this message sounds like the path is to glualint itself, however, it is to its config.

Reply:
We were able to remove this line alltogether